server/repository/src/migrations/v1_06_00/mod.rs

```diff
@@ -117,19 +118,52 @@ async fn migration_1_06_00() {
         }
     }
     #[derive(Queryable, Debug, PartialEq)]
-    pub struct StockMovementRow {
-        pub id: String,
-        pub item_id: String,
-        pub store_id: String,
-        pub quantity: i64,
-        pub datetime: NaiveDateTime,
+    struct StockMovementRow {
+        id: String,
+        item_id: String,
+        store_id: String,
+        quantity: i64,
+        datetime: NaiveDateTime,
+    }
+
+    table! {
+        requisition_line (id) {
+            id -> Text,
+            requisition_id -> Text,
+            item_link_id -> Text,
+            requested_quantity -> Integer,
+            suggested_quantity -> Integer,
+            supply_quantity -> Integer,
+            available_stock_on_hand -> Integer ,
+            average_monthly_consumption -> Integer,
+            snapshot_datetime -> Nullable<Timestamp>,
```

Comment:
For the future, I think we can leave Nullable out and actually, the requisition_line we are testing, only really needs to have id and item_link_id (when unless we are also using the data time to do upserting)